systemtest/src/test/java/io/strimzi/systemtest/KafkaST.java

```diff
@@ -1364,7 +1364,7 @@ void verifyVolumeNamesAndLabels(int kafkaReplicas, int diskCountPerReplica, int
     }
 
     @Test
-    void testPersistentStorageSize() throws Exception {
+    void testPersistentStorageSize() {
```

Comment:
It would make reviews of ST PRs lot easier if every PR didn't contain bunch of removals of `throws Exception`. Is this in any way related to the actual PR or is it just something you noticed while doing the PR? If it is related to the PR it is of course ok. But if not, please open one PR to fix all of these to make the reviews easier.

Reply:
Yeah, sorry for that...